packages/teleport/src/Kubes/Kubes.story.tsx

```diff
@@ -17,24 +17,42 @@ limitations under the License.
 import React from 'react';
 import { Kubes } from './Kubes';
 import { kubes } from './fixtures';
+import ConnectDialog from './ConnectDialog';
 
 export default {
   title: 'Teleport/Kubes',
 };
 
 export function Loaded() {
-  return <Kubes kubes={kubes} attempt={{ status: 'success' }} />;
+  return <Kubes {...props} attempt={{ status: 'success' }} />;
 }
 
 export function Loading() {
-  return <Kubes kubes={kubes} attempt={{ status: 'processing' }} />;
+  return <Kubes {...props} attempt={{ status: 'processing' }} />;
 }
 
 export function Failed() {
   return (
     <Kubes
-      kubes={kubes}
+      {...props}
       attempt={{ status: 'failed', statusText: 'server error' }}
     />
   );
 }
+
+export function Connect() {
+  return (
+    <ConnectDialog
+      onClose={() => null}
+      user={'sam'}
+      kubeName={'tele.logicoma.dev-prod'}
+    />
+  );
+}
+
+const props = {
+  kubes: kubes,
+  user: 'sam',
+  isEnterprise: true,
+  isLeafCluster: false,
+};
```

Comment:
Good job using props like this to avoid dups. But now storybook recently came out with a shorter, simpler way of rendering the same story with different props, so lets convert it to this new way of doing it: https://storybook.js.org/docs/react/writing-stories/args